src/docs/product/alerts-notifications/alert-settings.mdx

```diff
@@ -8,7 +8,9 @@ Configure alerts to your team's needs with alert rules and settings. Create new
 
 ## Rules
 
-The “Alert Rules” tab displays your alert rules, along with their current status, project, team, and date created. If you have [Performance Monitoring](/product/performance/) configured for your organization, the **Alerts** page also displays a “History” tab where you can find a list of triggered alerts, along with visual graphs, associated projects, and the last time the alert was triggered. In the “History” tab, the default view displays unresolved alerts.
+The “Alert Rules” tab displays your alert rules, along with their current status, project, team, and creation date. The list is filtered so that only alerts associated with the teams you're a member of are displayed. You can change this using the "Filter" button. The name of the button changes to reflect the number of filters applied, so when you first navigate to the page, it's labeled "2 Active Filters".
```

Comment:
```suggestion
The “Alert Rules” tab displays your alert rules, along with their current status, project, team, and creation date. By default, the list is filtered so that only alerts associated with the teams you're a member of, as well as ones that aren't associated with any team, are displayed. You can change this using the "Filter" button. The name of the button changes to reflect the number of filters applied, so when you first navigate to the page, it's labeled "2 Active Filters".
```

Reply:
Adding `by default` read a lot better to me? The information is accurate though :)